client/my-sites/stats/post-performance/style.scss

```diff
@@ -0,0 +1,7 @@
+.stats-post-performance__start-post {
+	padding-bottom: 26px;
+}
+
+.stats-post-performance__no-posts-message {
+	margin-bottom: 1em;
```

Comment:
Actually, there is a reason. This overrides a parent "em" margin and I found that using "ems" keeps the spacing good no matter the font-size of the message. 

That said, I can still use px if you prefer :).